Updated Microsoft Excel doc to add additional info on 1 min time limit (#6773)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud-integrations/semantic-layer/excel.md

<Tools 
type="Microsoft Excel"

> bullet_1="Results that take longer than one minute to load into Excel will fail. This time limit applies only to the loading process and does not include the time required for the data platform to run the query."

Reviewer: ```suggestion
bullet_1="Results that take longer than one minute to load into Excel will fail. This limit only applies to the loading process, not the time it takes for the data platform to run the query."
```
